.circleci/config.yml

```diff
@@ -434,6 +471,16 @@ jobs:
           command: |
             find . -type f -regex ".*/build/test-results/.*xml" -exec cp {} /tmp/workspace/reports/ \;
           when: always
+      - run:
+          name: Install the generated showcase library
+          command: |
+            cd workspace/gapic-generator/artman-genfiles/java
+            ./gradlew install -p gapic-google-cloud-showcase-v1alpha2
+      - run:
+          name: Run Showcase Integration Tests
+          command: |
+            cd workspace/gapic-generator/showcase/java
+            ./gradlew build test
```

Comment:
`./gradlew cleanTest check` might be better just in case... I'm not sure if the tests results would ever get cached in CI, but it can't hurt to clean it. Check would also verify the test code is formatted.

Reply:
Done!